packages/dds/tree/src/feature-libraries/sequence-field/utils.ts

```diff
@@ -666,17 +698,25 @@ export class DetachedNodeTracker {
         };
     }
 
-    private updateMark(
-        mark: Reattach<unknown>,
+    private updateMark<T>(
+        mark: Reattach<T>,
         revision: RevisionTag | undefined,
-        moveEffects: MoveEffectTable<unknown>,
+        moveEffects: MoveEffectTable<T>,
     ): void {
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
         const original = { rev: mark.detachedBy!, index: mark.detachIndex };
         const updated = this.getUpdatedDetach(original);
         if (updated.rev !== original.rev || updated.index !== original.index) {
             mark.detachedBy = updated.rev;
             mark.detachIndex = updated.index;
+            if (mark.type === "ReturnTo") {
+                getOrAddEffect(
+                    moveEffects,
+                    CrossFieldTarget.Source,
+                    mark.revision ?? revision,
+                    mark.id,
+                ).detacher = mark.detachedBy;
+            }
```

Comment:
Is this really needed? I removed it in #13706 so if you didn't re-introduce it then it's probably leftover from a bad git merge.